Integrations/integration-SCADAfence_CNM.yml

```diff
@@ -0,0 +1,467 @@
+description: fetching data from CNM
+configuration:
+- display: API auth secret
+  name: APISecret
+  defaultvalue: 
+  type: 4
+  required: true
+- display: API auth key
+  name: APIKey
+  defaultvalue: 
+  type: 4
+  required: true
+- display: API url
+  name: APIUrl
+  defaultvalue: https://172.28.128.10/externalApi
```

Comment:
why not to add `externalApi` in the integration itself?

Reply:
@AlexzxelA is the `externalApi` path constant? If so, perhaps we can set only the server URL here.

Reply:
it is sure possible

Reply:
@anara123 @AlexzxelA concatenating it to the API url now.